pkgs/applications/window-managers/awesome/default.nix

```diff
@@ -4,11 +4,17 @@
 , compton, procps, iproute, coreutils, curl, alsaUtils, findutils, xterm
 , which, dbus, nettools, git, asciidoc, doxygen
 , xmlto, docbook_xml_dtd_45, docbook_xsl, findXMLCatalogs
-, libxkbcommon, xcbutilxrm
+, libxkbcommon, xcbutilxrm, hicolor_icon_theme,
+#Hicolor is used for 4.0 themes such as CopyCat's awesome theme collection
+enableHicolorTheme ? false,
+
 }:
 
+assert enableHicolorTheme -> hicolor_icon_theme !=null;
+
 let
   version = "4.0";
+  available = x: x != null;
```

Comment:
@vcunat that is weird, because I didn't put that there, I don't really know what that would do :) but yea I removed it though